master/introduction/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: About Calico
+---
+
+Calico provides secure networking for containers and
+other distributed, dynamic workloads.
+
+Using the Linux kernel and BGP, Calico creates and manages 
+a flat virtual network, allowing any node to route to any other. 
+In private clouds, it can run without IP encapsulation or 
+network address translation for much faster speeds, easier 
+troubleshooting, and better interoperability.
+
+Using Calico's simple policy language, you can achieve
+fine-grained control over Layer 3 communications between
```

Comment:
Do we care about specifying Layer 3? Calico also offers L4 policy, for example.

Reply:
Yes, this will be removed in next iteration. Thanks for taking a look! I'll let you know when I have the revision.